examples/angular-cli/package.json

```diff
@@ -5,8 +5,7 @@
   "license": "MIT",
   "scripts": {
     "build": "ng build",
-    "prebuild-storybook": "yarn storybook:prebuild",
-    "build-storybook": "build-storybook -s src/assets",
+    "build-storybook": "yarn storybook:prebuild && build-storybook -s src/assets",
```

Comment:
Do note that script names containing `:` (colon) are global and thus don't allow duplicates
https://yarnpkg.com/getting-started/qa/#how-to-share-scripts-between-workspaces

Reply:
Good to know! I think we will need to do a big clean around that then.